bin/mff2json.py

```diff
@@ -0,0 +1,52 @@
+"""
+Copyright 2019 Brain Electrophysiology Laboratory Company LLC
+
+Licensed under the ApacheLicense, Version 2.0(the "License");
+you may not use this module except in compliance with the License.
+You may obtain a copy of the License at:
+
+http: // www.apache.org / licenses / LICENSE - 2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ANY KIND, either express or implied.
+"""
+
+"""
+This script is used to export the contents of an MFF file into a JSON format.
+
+@author: Damian Persico - persicodamian@gmail.com
+"""
+
+from os.path import splitext
+from mffpy import Reader, Writer
+from argparse import ArgumentParser
+
+
+def parse_args():
+    parser = ArgumentParser()
```

Comment:
unless we're planning on having more arguments in the future, since this is only used the one time and effectively is just two lines of code, I'd remove this function. Additionally, I'd move line 36-52. End would look like this:

```Python3.6
def mff2json(input_filename):
    # lines 36-52 go here

if __name__ == "__main__":
    parser = ArgumentParser()
    parser.add_argument("--mff_file", type=str, help="Path to an MFF directory")
    mff2json(parser.mff_file)
```